### PR TITLE
Fix "Cannot convert undefined to a BigInt"

### PR DESCRIPTION
**Pull request by KernelFreeze**
- [X] I have verified my changes didn't break the `example` project.

#### Description

I fixed the following exception when executing queries:

```
Uncaught (in promise) TypeError: Cannot convert undefined to a BigInt
  insertId: results.lastRowId !== null ? BigInt(results.lastRowId) : undefined,
                                         ^
      at executeQuery
```

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ class D1Connection implements DatabaseConnection {
     }
 
     return {
-      insertId: results.lastRowId !== null ? BigInt(results.lastRowId) : undefined,
+      insertId: !results.lastRowId ? undefined : BigInt(results.lastRowId),
```
Comment (aidenwallis) [inline src/index.ts:110]: Is this documented in cloudflare's docs anywhere, do you have a link to it? If so, lets insert a comment?

I'm also concerned about other zero values here, hence the explicit check

Comment (KernelFreeze) [reply src/index.ts:110]: It's not documented sadly, but I get a lot of `undefined`s

**Comment by aidenwallis** [comment]
🤔 Now I'm wondering if `0` should really be `BigInt(0)` here lol, perhaps `results.lastRowId === undefined || results.lastRowId === null ? undefined : BigInt(results.lastRowId)`

there is a difference between those two and we should aim to return the correct types

**Comment by KernelFreeze** [comment]
Is it ok like this?

**Review by aidenwallis — APPROVED** [review]

**Comment by aidenwallis** [comment]
These changes all make sense to me now, thank you for the contribution!

**Pull request merged**